### PR TITLE
fix: restart failed issue

### PR DESCRIPTION
**Pull request by azura27**
### What this PR does / why we need it:

Issue #3202 
refer to https://github.com/apache/apisix/pull/2869
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added? yes
* [ ] Have you added corresponding test cases? yes
* [ ] Have you modified the corresponding document? 
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list] (https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by spacewander** [comment]
```
Checking apisix/cli/ops.lua                       1 warning

    apisix/cli/ops.lua:390:101: line is too long (113 > 100)

Total: 1 warning / 0 errors in 127 files
Makefile:73: recipe for target 'lint' failed
make: *** [lint] Error 1
```
Need to fix the lint.

File: apisix/cli/ops.lua [hunk 1]
```diff
@@ -382,11 +382,19 @@ local function start(env, ...)
...
+            if res == "" then
+                print("nginx.pid exists with no Nginx process, file will be overwritten")
+            else
+                print("APISIX is running...")
+                return nil
+            end
+        else
+            print("failed to read the result of command: " .. lsof_cmd)
+            return nil
         end
```
Comment (spacewander) [inline apisix/cli/ops.lua:398]: We can put all `return nil` after here?

**Comment by membphis** [comment]
ping @azura27

**Comment by Yiyiyimu** [comment]
I think the issue got fixed by #3416. @azura27 plz have a check then we could close this PR

**Comment by azura27** [comment]
OK, got it

**Pull request closed without merging**